app/views/backend/cash_transfers/_form.html.haml

```diff
@@ -5,4 +5,5 @@
   = f.input :currency_rate
   = f.referenced_association :reception_cash
   = f.input :description, field: :text
+  = f.association :creator, input_html: { value: current_user.id }, as: :hidden
```

Comment:
Risky. If we need this, it needs to be moved up into the controller.
Also: don't timestamps already handle this?